### PR TITLE
More set block state flags

### PR DESCRIPTION
**Pull request by Earthcomputer**
Flags! :black_flag: :white_flag: :rainbow_flag:

### CODE REVIEW

File: unpick-definitions/set_block_state_flags.unpick [hunk 1]
```diff
@@ -1,5 +1,6 @@
 v2
 
+#flag set_block_state_flags net/fabricmc/yarn/constants/SetBlockStateFlags DEFAULT
```
Comment (Earthcomputer) [inline unpick-definitions/set_block_state_flags.unpick]: Unpick doesn't seem to handle this very well yet, so I commented this out for now.

Comment (modmuss50) [reply unpick-definitions/set_block_state_flags.unpick]: Humm, yeah.. Might need to take a look at unpick to handle this.

**Review by liach — APPROVED** [review]

**Comment by liach** [comment]
Just curious, is there any tool to modify unpick format easily, or is it usually modified by hand like access wideners?

**Comment by Earthcomputer** [comment]
I did it by hand, hopefully there will be tools in Enigma in the near future.

**Comment by liach** [comment]
Just a side note: I will put new/refactor on unpick prs based on how they change unpick definitions. In fact, I wonder how bad it is for mapping users if a constant holding utility class is changed. Does it cause more incompatibility than regular mapping refactors?

**Comment by Earthcomputer** [comment]
The unpick issue affecting `SetBlockStateFlags.DEFAULT` is fixed by https://github.com/FabricMC/unpick/pull/3, going to re-enable the DEFAULT flag in this PR so that it works once unpick is fixed.

**Review by YanisBft — APPROVED** [review]
Works well

**Comment by liach** [comment]
@Earthcomputer Feel free to revert #2189 in your changes. I think your patch fixed the spamming problem of unpick where it logs an info message for every method it visits, but it might need some testing to see if the log spam is actually gone.

**Review by modmuss50 — APPROVED** [review]

**Comment by Earthcomputer** [comment]
@liach I didn't remove that log spam

**Comment by liach** [comment]
Turns out the muting of unpick is still needed, sadly. Yeah I tested locally as well and the log spam is still there, but otherwise things still appear good :+1:

**Pull request merged**